docs/topics/whatsnew17.md

```diff
@@ -0,0 +1,1046 @@
+  and [builder inference](#stable-builder-inference).
+* [There's now an underscore operator for type args](#underscore-operator-for-type-arguments). You can use it to
+  automatically infer a type of argument when other types are specified.
+* [This release allows implementation by delegation to an inlined value of an inline class](#allow-implementation-by-delegation-to-an-inlined-value-of-an-inline-class). You can now create
+  lightweight wrappers that do not allocate memory in most cases.
+
+You can also find a short overview of the changes in this video:
+
+<video href="54WEfLKtCGk" title="What's new in Kotlin 1.7.0"/>
+
+## New Kotlin K2 compiler for the JVM in Alpha
+
+This Kotlin release introduces the **Alpha** version of the new Kotlin K2 compiler. The new compiler aims to speed up
+the development of new language features, unify all of the platforms Kotlin supports, bring performance improvements, and
+provide an API for compiler extensions.
+
+We've already published some detailed explanations of our new compiler and its benefits:
+
+* [The Road to the New Kotlin Compiler](https://www.youtube.com/watch?v=iTdJJq_LyoY)
+* [K2 Compiler: a Top-Down View](https://www.youtube.com/watch?v=db19VFLZqJM)
+
+It's important to point out that with the Alpha version of the new K2 compiler we were primarily focused on performance
+improvements, and it only works with JVM projects. It doesn't support Kotlin/JS, Kotlin/Native, or other multi-platform
+projects, and none of compiler plugins, including [kapt](kapt.md), work with it.
+
+Our benchmarks show some outstanding results on our internal projects:
+
+| Project       | Current Kotlin compiler performance | New K2 Kotlin compiler performance | Performance boost |
+|---------------|-------------------------------------|------------------------------------|-------------------|
+| Kotlin        | 2.2 KLOC/s                          | 4.8 KLOC/s                         | ~ x2.2            |
+| YouTrack      | 1.8 KLOC/s                          | 4.2 KLOC/s                         | ~ x2.3            |
+| IntelliJ IDEA | 1.8 KLOC/s                          | 3.9 KLOC/s                         | ~ x2.2            |
+| Space         | 1.2 KLOC/s                          | 2.8 KLOC/s                         | ~ x2.3            |
+
+> The KLOC/s performance numbers stand for the number of thousands of lines of code that the compiler processes per
+> second.
+>
+> {type="tip"}
+
+You can check out the performance boost on your JVM projects and compare it with the results of the old compiler. To
+enable the Kotlin K2 compiler, use the following compiler option:
+
+```bash
+-Xuse-k2
+```
+
+Also, the K2 compiler [includes a number of bugfixes](https://youtrack.jetbrains.com/issues/KT?q=tag:%20fixed-in-frontend-ir%20sort%20by:%20Priority,%20votes,%20updated).
```

Comment:
This link appears to currently result in:

> The search query contains unknown values
The value "fixed-in-frontend-ir" isn't used for the tag field.

Reply:
Hey Ian,
The correct tag is `FIR-preview-qa`. I fixed the link in https://github.com/JetBrains/kotlin-web-site/pull/3326
Thanks for noticing and your help!